packages/wonder-blocks-dropdown/components/multi-select.test.js

```diff
@@ -50,16 +57,76 @@ describe("MultiSelect", () => {
         opener.simulate("mouseup");
         opener.simulate("click");
         expect(select.state("open")).toEqual(true);
+        expect(onOpen).toHaveBeenCalledTimes(1);
+        expect(onClose).toHaveBeenCalledTimes(0);
+
+        // Close select with mouse
+        opener.simulate("mousedown");
+        opener.simulate("mouseup");
+        opener.simulate("click");
+        expect(select.state("open")).toEqual(false);
+        expect(onOpen).toHaveBeenCalledTimes(1);
+        expect(onClose).toHaveBeenCalledTimes(1);
+
+        // Open select with mouse
```

Comment:
This should probably be a separate test case, something along the line of `it("should re-open when clicking after closing", () => {})` or something like that.